templates/control/edit_commissionprices.html

```diff
@@ -1,169 +1,268 @@
-$def with (query)
-$:{RENDER("common/stage_title.html", ["Edit Commission Prices", "Settings"])}
+$def with (query, currencies, presets, profile)
+$code:
+  def _SELECTED(x, y):
+    return ' selected="selected"' if profile['settings'][x] == y else ''
 
+$:{RENDER("common/stage_title.html", ["Edit Commission Prices", "Settings"])}
 <div id="commishprices-content" class="content form clear">
-
-  <div class="column classes">
-    <h3>Create Classification</h3>
-
-    <form name="controlcreatecommishclass" action="/control/createcommishclass" method="post" class="clear">
-      $:{CSRF()}
-      <label for="commishprices-addclass">Class Title</label>
-      <input type="text" id="commishprices-addclass" class="input last-input" name="title" required="required" placeholder="ex: 'Sketches', 'Badges'" />
-      <button type="submit" class="button last-input positive" style="float: right;">Create Class</button>
-    </form>
-
-    $if(query['class']):
-      <h3>Edit Classification</h3>
-      <form name="controleditcommishclass" action="/control/editcommishclass" method="post" class="clear">
-        $:{CSRF()}
-        <label for="commishprices-editclass">Class Title</label>
-        <select id="commishprices-editclass" class="input" name="classid">
-          <option value="">Select class:</option>
+  <div>
+    <h3>My General Commission Info</h3>
+    <p>Information about commissioning you and preferences for commission content</p>
+    <div class="commish-row">
+      <div class="info">
+        <form name="controleditcommishtext" action="/control/editcommishinfo" method="post" class="clear">
+          $:{CSRF()}
+          <!-- Copied from edit_profile.html. TODO(hyena): Only put this in one place, probably here. -->
+          <label for="editprofile-setcommish">Commissions</label>
+          <select id="editprofile-setcommish" class="input" name="set_commish">
+            <option value="o"${_SELECTED(0, 'o')}>I am currently accepting commissions</option>
+            <option value="s"${_SELECTED(0, 's')}>I may sometimes accept commissions</option>
+            <option value="f"${_SELECTED(0, 'f')}>My commission queue is currently filled</option>
+            <option value="c"${_SELECTED(0, 'c')}>I am not accepting commissions right now</option>
+            <option value="e"${_SELECTED(0, 'e')}>This is not applicable to me</option>
+          </select>
+
```

Comment:
It might be nice if at this location in the control panel, there's a comment like 'Setting your status to X will make commission items on this page appear in the [marketplace].